Integration Tests Periodically fail due to "Recursive Update"

##### **Overview of the issue**

The integration tests are periodically failing due to the following:

```
[ERROR] testCorsFilterDeactivated  Time elapsed: 2.944 s  <<< ERROR!
io.micronaut.context.exceptions.BeanInstantiationException: 
Error instantiating bean of type  [io.micronaut.validation.validator.resolver.CompositeTraversableResolver]

Message: Recursive update
Path Taken: new DefaultValidator([ValidatorConfiguration configuration]) --> DefaultValidatorConfiguration.setTraversableResolver([TraversableResolver traversableResolver]) --> new CompositeTraversableResolver([List traversableResolvers])
	at com.mycompany.myapp.config.CorsTest.testCorsFilterDeactivated(CorsTest.java:56)
Caused by: java.lang.IllegalStateException: Recursive update
	at com.mycompany.myapp.config.CorsTest.testCorsFilterDeactivated(CorsTest.java:56)
```

##### **Reproduce the error**

https://github.com/jhipster/generator-jhipster-micronaut/runs/886184785?check_suite_focus=true

##### **Project configuration**

All of the sample projects seem to periodically have this issue

-   [x] Checking this box is mandatory (this is just to show you read everything)

--- a/generators/constants.js
+++ b/generators/constants.js
@@ -25,11 +25,11 @@ module.exports = {
         postgresql: 'org.postgresql.Driver',
     },
     versions: {
-        micronaut: '2.0.0',
-        micronautData: '1.1.1',
+        micronaut: '2.0.1',
+        micronautData: '1.1.3',
 
         hibernate: '5.4.17.Final',
-        jackson: '2.11.0',
+        jackson: '2.11.1',
         javassist: '3.24.0-GA', // Should match Hibernate deps
         javaxMail: '1.6.2',
         jbcrypt: '0.4',

--- a/generators/server/templates/build.gradle.ejs
+++ b/generators/server/templates/build.gradle.ejs
@@ -332,7 +332,7 @@ dependencies {
     implementation "org.hibernate:hibernate-jcache"
         <%_ } _%>
     <%_ } _%>
-    implementation "io.micronaut.cache:micronaut-cache-core:$micronaut_version"
+    implementation "io.micronaut.cache:micronaut-cache-core"
     <%_ if (['ehcache', 'caffeine', 'hazelcast', 'infinispan', 'redis'].includes(cacheProvider) || applicationType === 'gateway') { _%>
     implementation "javax.cache:cache-api"
     <%_ } _%>

